backend/internal/domain/helpers/password_hasher.go

```diff
@@ -0,0 +1,26 @@
+package helpers
+
+import (
+	"errors"
+
+	"golang.org/x/crypto/bcrypt"
+)
+
+const (
+	DefaultCost int = 10
+)
+
+func HashPassword(password string) (string, error) {
+	if len(password) > 36 {
+		ErrPasswordTooLong := errors.New("password length should not exceed 72 bytes")
+		return "", ErrPasswordTooLong
+	}
+
+	bytes, err := bcrypt.GenerateFromPassword([]byte(password), DefaultCost)
```

Comment:
Usually, when you return an error, you should return a zero value of a type. The approach is to make value useless if an error occurred.

Here, I think the best approach would be to say:
```go
if err != nil {
    return "", err
}
return string(bytes), nil
```